docs(35/centos): Suggested updates to CentOS Guide PR #1367 (#1385)
Kong/docs.konghq.com: app/enterprise/0.35-x/deployment/installation/centos.md

intended only as an example -- you will want to modify and take additional
measures to secure your Kong Enterprise system before moving to a production
environment.

## Prerequisites

To complete this guide you will need:

- A CentOS 7 system with root equivalent access.
- The ability to SSH to the system.

## Step 1. Download Kong Enterprise

1. Option 1. Download via **Packages**

    Log in to [Bintray](http://bintray.com) to download the latest Kong 
    Enterprise RPM for CentOS 7. Your **Sales** or **Support** contact will
    email this credential to you.
    
    Copy the file to your home directory:

    ```
    $ scp kong-enterprise-edition-0.35-1.el7.noarch.rpm <centos user>@192.168.91.109:~
    ```

2. Option 2. Download via **YUM**

    Edit the `baseurl` field in the yum repository file `/etc/yum.repos.d` :

    ```
    baseurl=https://<USERNAME>:<API_KEY>@kong.bintray.com/kong-enterprise-edition-rpm/centos/$releasever
    ```
    Replace `<USERNAME>` with your Bintray account information

    Set `$releasever` to the correct CentOS version (e.g. `6` or `7`)

3. Obtain your Kong Enterprise license

>    If you do not already have your license file, it can be downloaded from your

Reviewer: ```suggestion
   If you do not already have your license file, you can download it from your
```

   ```
4. Securely copy the license file to the CentOS system

    ```
    $ scp kong-se-license.json <centos username>@192.168.91.109:~
    ```

## Step 2. Install Kong Enterprise

1. Install Kong Enterprise

    ```
    $ sudo yum install kong-enterprise-edition-0.35-1.el7.noarch.rpm
    ```
  >Note: Your version may be different based on when you obtained the rpm

2. Copy the license file to the `/etc/kong` directory

    ```
    $ sudo cp kong-se-license.json /etc/kong/license.json
    ```
    Kong will look for a valid license in this location.

## Step 3. Setup PostgreSQL

1. Install PostgreSQL

    ```
    $ sudo yum install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm

    $ sudo yum install postgresql95 postgresql95-server
    ```

2. Initialize the PostgreSQL Database

    ```
    $ sudo /usr/pgsql-9.5/bin/postgresql95-setup initdb
    ```

> 3. Start PostgresQL and Enable Automatic Start

Reviewer: ```suggestion
3. Start PostgreSQL and Enable Automatic Start
```

    ```

## Step 4. Create a Kong database and user

1. Switch to PostgreSQL user

    ```
    $ sudo -i -u postgres
    ```

2. Launch PostgreSQL

    ```
    $ psql
    ```

3. Run the following command to:

    -- Create a Kong user and database

    --  Make Kong the owner of the database

    --  Set the password of the Kong user to 'kong'

    ```
    $ CREATE USER kong; CREATE DATABASE kong OWNER kong; ALTER USER kong WITH password 'kong';
    ```

    >⚠️**Note**: Make sure the username and password for the Kong Database are
    >kept safe. We have used a simple example for illustration purposes only.

4. Exit from PostgreSQL

    ```
    $ \q
    ```

5. Return to terminal

    ```
    $ exit
    ```

6. Run the following command to access the PostgreSQL configuration file.

    ```
    $ sudo vi /var/lib/pgsql/9.5/data/pg_hba.conf
    ```

7. Under IPv4 local connections replace `ident` with `md5`

    | TYPE | DATABASE | USER | ADDRESS      | METHOD |
    | # IPv4 local connections:                      |
    | host | all      | all  | 127.0.0.1/32 | **md5**|
    | # IPv6 local connections:                      |
    | host | all      | all  | ::1/128      | ident  |

> 8. Restart PostgresQL

Author: ```suggestion
8. Restart PostgreSQL
```

    ```
    $ sudo systemctl restart postgresql-9.5
    ```

## Step 5. Modify Kong's configuration file

To use the newly provisioned PostgreSQL database, Kong's configuration file
must be modified to accept the correct PostgreSQL user and password.

1. Make a copy of the default configuration file

    ```
    $ cp /etc/kong/kong.conf.default /etc/kong/kong.conf
    ```

> 2. Uncomment and update the PostgresQL database properties inside the Kong conf:

Author: ```suggestion
2. Uncomment and update the PostgreSQL database properties inside the Kong conf:
```

    ```
    $ sudo vi [/path/to/kong.conf]
    ```
    ```
    pg_user = kong
    pg_password = kong
    pg_database = kong
    ```

> ## Step 6. Seed the **Super Admin** *(optional)*

Reviewer: bold doesn't render in a title, italics will. This suggestion assumes that (optional) was the point to emphasize.

```suggestion
## Step 6. Seed the Super Admin _(optional)_
```

> For added security and Role-Based Access Control (RBAC), it is best to seed 

Reviewer: ```suggestion
For the added security of Role-Based Access Control (RBAC), it is best to seed 
```

> the **super-admin** before initial start up.

Reviewer: ```suggestion
the **Super Admin** before initial start-up.
```

> Create an environment variable with the desired **super-admin** password:

Reviewer: ```suggestion
Create an environment variable with the desired **Super Admin** password:
```

    $ export KONG_PASSWORD=<password-only-you-know>

> This will be used during migrations to seed the initial **super-admin** 

Reviewer: ```suggestion
This will be used during migrations to seed the initial **Super Admin** 
```

password within Kong.

## Step 7. Start Kong

1. Run migrations to prepare the Kong database

    ```
    $ kong migrations bootstrap -c /etc/kong/kong.conf
    ```

2. Start Kong

    ```
    $ sudo /usr/local/bin/kong start -c /etc/kong/kong.conf
    ```

3. Verify Kong is working

    ```
    curl -i -X GET --url http://localhost:8001/
    ```

## Troubleshooting

If you did not receive an HTTP/1.1 200 OK message, or need assistance completing
setup reach out to your **Support contact** or head over to the
[Support Portal](https://support.konghq.com/support/s/).

## Next Steps

> Work through Kong's series of 

Reviewer: ```suggestion
Work through Kong Enterprise's series of 
```

> [Getting Started](/enterprise/latest/getting-started) guides to get the most

Reviewer: Why use `latest` instead of `{{kong.page_version}}`?

Author: Just a future-proofing decision incase it doesn't get updated each release
